sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -167,7 +167,6 @@ export class ServiceBusClient {
 
 // @public
 export interface ServiceBusClientOptions {
-    dataTransformer?: DataTransformer;
```

Comment:
Line 39: DataTransformer is still being exported

Reply:
Thanks for memorizing the whole API, removed now :)

Reply:
hehe